pkg/playbook/app/installer/crypto.go

```diff
@@ -27,11 +28,10 @@ import (
 	"strings"
 	"time"
 
-	"go.uber.org/zap"
-
 	"github.com/Venafi/vcert/v5/pkg/certificate"
 	"github.com/Venafi/vcert/v5/pkg/playbook/app/vcertutil"
 	"github.com/Venafi/vcert/v5/pkg/util"
+	"go.uber.org/zap"
```

Comment:
Move zap dependency back to the previous block. application dependencies should be last

Reply:
Done by latest commit